CONTRIBUTING.md

```
@@ -11,51 +11,53 @@ Thanks for helping improve Actual-MoneyMoney! This guide explains how to get set
 ## Local Setup
 
 1. Clone the repository and install dependencies:
-   ```bash
-   git clone https://github.com/1cu/actual-moneymoney.git
-   cd actual-moneymoney
-   npm install
-   ```
+    ```bash
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)